src/dappsExplorer/DAppsExplorerScreenSearchFilter.tsx

```diff
@@ -168,10 +199,6 @@ export function DAppsExplorerScreenSearchFilter() {
             }
             ListHeaderComponent={
               <>
-                <DescriptionView
```

Comment:
is it in the task to remove this?

Reply:
Checking with @nityarao if we want this, otherwise I am fine reverting this change.

Reply:
fwiw i think it should be the other way around 🙈 we should be asked for the changes and then make them, rather than make the changes and revert them if we don't need them....